packages/neuvector/generated-changes/overlay/app-readme.md

```diff
@@ -0,0 +1,14 @@
+### Run-Time Protection Without Compromise
+
+NeuVector delivers a complete run-time security solution with container process/file system protection and vulnerability scanning combined with the only true Layer 7 container firewall. Protect sensitive data with a complete container security platform.
+
+NeuVector integrates tightly with Rancher and Kubernetes to extend the built-in security features for applications that require defense in depth. Security features include:
+
++ Build phase vulnerability scanning with Jenkins plug-in and registry scanning
++ Admission control to prevent vulnerable or unauthorized image deployments using Kubernetes admission control webhooks
++ Complete run-time scanning with network, process, and file system monitoring and protection
++ The industry's only layer 7 container firewall for multi-protocol threat detection and automated segmentation
++ Advanced network controls including DLP detection, service mesh integration, connection blocking and packet captures
++ Run-time vulnerability scanning and CIS benchmarks
+
+Please Note: Before installing this chart, you will need to get an image pull secret and license key from NeuVector. Without this data supplied, the chart will not work. Configure correct container runtime and runtime path.
```

Comment:
Is this true ? I thought the open-source version worked out of the box. If so, where can they get this from ? 

Also the image pull secret would be invalid because images moved to system_default_registry. Looks like it was removed below.

Reply:
Yes, mistake.  we will remove the pull secret requirement line from the app-readme.

Reply:
The license key too ?

Reply:
Yes, license key requirements will also be removed. Shall I update the PR with all these changes now or wait for more reviews from others too?

Reply:
Either works, typically people just push updates consistently

Reply:
I have updated the changes except for this comment https://github.com/rancher/charts/pull/1732#discussion_r848901197